tests/sqlparser_snowflake.rs

```diff
@@ -2483,6 +2483,37 @@ fn test_sf_trailing_commas() {
     snowflake().verified_only_select_with_canonical("SELECT 1, 2, FROM t", "SELECT 1, 2 FROM t");
 }
 
+#[test]
+fn test_sf_trailing_commas_in_from() {
+    snowflake().verified_only_select_with_canonical("SELECT 1, 2 FROM t,", "SELECT 1, 2 FROM t");
+}
+
+#[test]
+fn test_sf_trailing_commas_multiple_sources() {
+    snowflake()
+        .verified_only_select_with_canonical("SELECT 1, 2 FROM t1, t2,", "SELECT 1, 2 FROM t1, t2");
+}
+
+#[test]
+fn test_from_trailing_commas_with_limit() {
+    let sql = "SELECT a, FROM b, LIMIT 1";
```

Comment:
Can we add a similar scenario for the other entries in the allowed keyword list? (i.e. INTO, HAVING, WHERE)